docs/source/en/model_doc/gpt2.md

```diff
@@ -60,6 +60,73 @@ This model was contributed by [thomwolf](https://huggingface.co/thomwolf). The o
 - Enabling the *scale_attn_by_inverse_layer_idx* and *reorder_and_upcast_attn* flags will apply the training stability
   improvements from [Mistral](https://github.com/stanford-crfm/mistral/) (for PyTorch only).
 
+## Usage example
```

Comment:
We should have the equivalent added for decision transformer too

Reply:
See message above